examples/helloworld/package.json

```diff
@@ -43,9 +43,17 @@
     "lodash": "^4.17.5",
     "mobx": "^4",
     "mobx-react": "^5.4.3",
-    "serverless-http": "^1.8.0"
+    "serverless-http": "^1.8.0",
+    "react-chartjs-2": "^2.7.6",
+    "chart.js": "^2.8.0"
   },
   "seagull": {
     "domains": []
+  },
+  "vendorBundleIncludes": {
+    "add": [
```

Comment:
- please add this as a key under seagull, not top level
- add as a key looks weird...

Reply:
- done
- handling the vendor ingredients in seagull projects would imho violate SOC and make it necessary to know about and adapt to changes in seagull dependencies within every seagull project. i get your point concerning the simplicity though ...